Plugin/DotpayDirectPlugin.php

```diff
@@ -90,6 +90,9 @@ class DotpayDirectPlugin extends AbstractPlugin
     /** @var bool */
     protected $expirationTime;
 
+    /** @var bool */
```

Comment:
it's a nullable string, based on what you've set in `Configuration`
```suggestion
    /** @var string|null */
```